packages/additionalapi/src/controllers/AdditionalApi.ts

```diff
@@ -22,6 +22,11 @@ export class AdditionalApiImpl implements AdditionalApi {
         }
         logger.dbLog("API.getNodeTree is " + idList)
         const result = await this.context.additionalApiWorker.getNodeTree(idList, depthLimit)
+        lionwebResponse(res, 200, {
```

Comment:
It may be useful add in the README a description of when we will use the HTTP code to signal errors and when instead we will use the success flag to do so